ssl/statem/extensions_clnt.c

```diff
@@ -108,8 +108,8 @@ static int use_ecc(SSL *s)
     return i < end;
 }
 
-int tls_construct_ctos_ec_pt_formats(SSL *s, WPACKET *pkt, X509 *x,
-                                     size_t chainidx, int *al)
+int tls_construct_ctos_ec_pt_formats(SSL *s, WPACKET *pkt, unsigned int context,
```

Comment:
we now have SIX parameters on every parse function.  Time to make a custom struct that gets passed in I think.